src/sentry/models/organization.py

```diff
@@ -276,3 +277,21 @@ def send_delete_confirmation(self, audit_log_entry, countdown):
             type='org.confirm_delete',
             context=context,
         ).send_async([o.email for o in owners])
+
+    def suspend(self, **kwargs):
+        assert not self.flags.suspended
+
+        self.flags.suspended = True
+        self.save()
+
+        if 'owner_id' not in kwargs:
+            kwargs['owner_id'] = self.get_default_owner().id
+
+        if 'owner_email' not in kwargs:
```

Comment:
this is a bit odd, as if you pass owner_id but not owner_email you'll get a mismatch

I'd probably just have them pass owner, and do the rest from within this logic